GameServerConsole/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using CommandLine;
+using GameServerConsole.Properties;
+using LeagueSandbox.GameServer;
+using LeagueSandbox.GameServer.Logging;
+using LeagueSandbox.GameServerConsole.Utility;
+
+namespace LeagueSandbox.GameServerConsole
+{
+    internal class Program
+    {
+        private static ILogger _logger;
+
+        private static void Main(string[] args)
+        {
+            _logger = LoggerProvider.GetLogger();
+
+            ArgsOptions.Parse(args).WithParsed(parsedArgs =>
+            {
+                parsedArgs.GameInfoJson = LoadConfig(
+                    parsedArgs.GameInfoJsonPath,
+                    parsedArgs.GameInfoJson,
+                    Encoding.UTF8.GetString(Resources.GameInfo));
+
+                var gameServerBlowFish = "17BLOhi6KZsTtldTsizvHg==";
+                var gameServerLauncher = new GameServerLauncher(
+                    parsedArgs.ServerPort,
+                    parsedArgs.GameInfoJson,
+                    gameServerBlowFish);
+
+#if DEBUG
+                var configGameServerSettings = GameServerConfig.LoadFromJson(LoadConfig(
+                    parsedArgs.GameServerSettingsJsonPath,
+                    parsedArgs.GameServerSettingsJson,
+                    Encoding.UTF8.GetString(Resources.GameServerSettings)));
+                if (configGameServerSettings.AutoStartClient)
+                {
+                    var leaguePath = configGameServerSettings.ClientLocation;
+                    if (Directory.Exists(leaguePath))
+                    {
+                        leaguePath = Path.Combine(leaguePath, "League of Legends.exe");
+                    }
+                    if (File.Exists(leaguePath))
+                    {
+                        var startInfo = new ProcessStartInfo(leaguePath)
+                        {
+                            Arguments = String.Format("\"8394\" \"LoLLauncher.exe\" \"\" \"127.0.0.1 {0} {1} 1\"",
```

Comment:
You could use string interpolation for that

Reply:
True. This line is unchanged. It’s green highlighted because git is weird. 
Assume most GameServerConsole changes aren’t actual changes.